hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OMDBUpdatesHandler.java

```diff
@@ -268,6 +268,12 @@ public void markCommit(byte[] bytes) throws RocksDBException {
      */
   }
 
+  @Override
+  public void markCommitWithTimestamp(byte[] bytes, byte[] bytes1)
+      throws RocksDBException {
```

Comment:
```suggestion
  public void markCommitWithTimestamp(final byte[] xid, final byte[] ts)
      throws RocksDBException {
    /**
     * There are no use cases yet for this method in Recon. These will be
     * implemented as and when need arises.
     */
```

I'd suggest removing the `@Override` annotation to improve compatiblity when compiling with an older RocksDB JNI just in case.

I tested with 7.0.2 previously in HDDS-6511. Without the annotation it works as well. Should be the same for 7.0.3

Reply:
@kerneltime Just a thought - If you are not already, you might also like to try switching to the ByteBuffer versions of the methods that move bytes around (as opposed to the older `byte[]` based versions). Several people have reported better performance with the newer API

Reply:
@adamretter thank you, will look into this.